dotcom-rendering/src/paletteDeclarations.ts

```diff
@@ -2976,7 +2989,7 @@ const articleBackgroundLight: PaletteFunction = ({
 		case ArticleDesign.FullPageInteractive:
 			return 'transparent';
 		case ArticleDesign.Gallery:
-			return sourcePalette.neutral[7];
+			return '#0d0d0d';
```

Comment:
In frontend the colour for the left and right columns in wide breakpoint, is chosen based on the existing background colour `rgba(0, 0, 0, 0.25)`. 

Because we have switched to using padded container, we no longer have the direct background edges exposed to create the overlay effect. Therefore, I'm using the `#0d0d0d` in light mode and `#131313` in dark mode. 
- light mode: rgba(0, 0, 0, 0.25)  overlaid on top of the #121212 => #0d0d0d
- light mode: rgba(0, 0, 0, 0.25)  overlaid on top of the #1A1A1A => #131313